src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
+server.port = 8084
 
 spring.datasource.url=jdbc:mysql://localhost:3306/meetup?useSSL=false&serverTimezone=UTC&useLegacyDatetimeCode=false&allowPublicKeyRetrieval=true&useSSL=false
```

Comment:
Hasła i dane wrażliwe lepiej przez zmienne środowiskowe.

Drugi problem z tym jest też taki, że wymuszamy jakie hasło i namiar na bazę będą mieli wszyscy w projekcie.

Proponuję zerknąć jak to jest zrobione na tym przykłądowym repo https://github.com/towczare/HerokuDemo